library/websphere-liberty

```diff
@@ -2,35 +2,62 @@ Maintainers: Wendy Raschke <wraschke@us.ibm.com> (@wraschke),
              Andy Naumann <naumann@us.ibm.com> (@naumanna),
              Arthur De Magalhaes <arthurdm@ca.ibm.com> (@arthurdm)
 GitRepo: https://github.com/WASdev/ci.docker.git
-GitCommit: d581464c90b50722f1aaa59357cc15648a7f1313
+GitCommit: c1295e2e6ad436f143635013153cede9893c4475
 Architectures: amd64, i386, ppc64le, s390x
 
-Tags: 18.0.0.3-javaee8, javaee8, latest
+Tags: beta
+Directory: beta
+
+Tags: 18.0.0.4-javaee8, javaee8, latest
+Directory: ga/18.0.0.4/javaee8
+
+Tags: 18.0.0.4-webProfile8, webProfile8
+Directory: ga/18.0.0.4/webProfile8
+
+Tags: 18.0.0.4-microProfile1, microProfile1
+Directory: ga/18.0.0.4/microProfile1
+
+Tags: 18.0.0.4-microProfile2, microProfile2
+Directory: ga/18.0.0.4/microProfile2
+
+Tags: 18.0.0.4-springBoot2, springBoot2
+Directory: ga/18.0.0.4/springBoot2
+
+Tags: 18.0.0.4-kernel, kernel
+Directory: ga/18.0.0.4/kernel
+
+Tags: 18.0.0.4-springBoot1, springBoot1
+Directory: ga/18.0.0.4/springBoot1
+
+Tags: 18.0.0.4-webProfile7, webProfile7
+Directory: ga/18.0.0.4/webProfile7
+
+Tags: 18.0.0.4-javaee7, javaee7
+Directory: ga/18.0.0.4/javaee7
+
+Tags: 18.0.0.3-javaee8
```

Comment:
Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
hi @tianon - we would like to keep the N-2 versions updated going forward, as customers need a bit of time to move up the versions, so we wanted those versions to stay current with the base changes, etc.